02_analysis_scripts/README.md

```diff
@@ -0,0 +1,32 @@
+## Analysis
+Below are exmaples of how to use analysis scripts to analyse the dataset for F@H data in [DOI link]. It is recommened to run each of these scripts in separate directories.
+
+**Note**: these scripts were developed as part of a time sensive project, as a result they are not memory efficient.
+
+### Contact analysis
+`analyze_antibody_rbd_contacts.py` can be used to generate a series of `.pkl` files containing an N x M x No. frames array of distances between heavy atoms at the RBD:Ab interface.
+
+For example, analysing the F@H `17341` project (S2H97:Ab):
+
+```python
+python analyze_antibody_rbd_contacts.py -mutant WT -pdb_file system.pdb -traj_path path/to/traj/ -fah_project_code 17341 -chain_info_file chain_info_S2H97_RBD.yaml -run_number 0 -discard_time 100
```

Comment:
can you define what `-chain_info_file` and `-run_number` and `-discard_time` correspond to?

Reply:
And also specify that there are 5 RUNs for each project, so we ran this command on each of the 5 RUNs separately. And that we used a discard time of 100 for all analysis jobs.